ChaosMod/Effects/db/Vehs/VehsFuelSpill.cpp

```diff
@@ -0,0 +1,15 @@
+/*
+	Effect by Joshuax8
+*/
+
+#include <stdafx.h>
+
+static void OnTick()
+{
+	for (auto veh : GetAllVehs())
+	{
+		SET_VEHICLE_PETROL_TANK_HEALTH(veh, 0.f);
```

Comment:
You could run [`SET_VEHICLE_FUEL_LEVEL`](https://docs.fivem.net/natives/?_0xBA970511) here as well, to ensure it never runs out

Reply:
Thanks for the suggestion! Will have to do some testing how long it takes to run out and if this is needed (once it's working).

Reply:
`SET_VEHICLE_FUEL_LEVEL` seems to be unavailable in the bundled `vendor/scripthookv/inc/natives.h` unfortunately.

Reply:
`SET_VEHICLE_FUEL_LEVEL` is a FiveM exclusive (not in game by default) so unable to use it because of that (thanks pongo for the hint).